cmake/options.cmake

```
@@ -11,6 +11,7 @@ option(sleigh_ENABLE_EXAMPLES "Set to true to build examples" ON)
 option(sleigh_ENABLE_DOCUMENTATION "Set to true to enable the documentation")
 option(sleigh_ENABLE_PACKAGING "Set to true to enable packaging")
 option(sleigh_ENABLE_SANITIZERS "Set to true to enable sanitizers")
+option(remill_ENABLE_PATCHES "Enables remill patches" OFF)
```

```suggestion
option(sleigh_remill_ENABLE_PATCHES "Enables remill patches" OFF)
```
Nit: I'd prefer the option to be prefixed with _this_ project's name. It's for easier discoverability and like a namespace to hopefully avoid clobbering variables from remill